src/utils/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
```

Comment:
It is a good practice to name color in this pattern `color-blue-accent`, you could modify those variables names